style.css

```diff
@@ -272,6 +273,10 @@ nav .logo {
   color: #253858;
 }
 
+.modelContainer img {
+  max-width: 100%;
+}
```

Comment:
## BEAUTIFUL DESIGN, GOOD JOB 👏
- [ ] [OPTIONAL] Please make the popup image `width: 100%;` to make it look better. Somehow `max-width` not working well. GOOD LUCK 🍀👍

Reply:
Okay, thanks.  I will make this changes and thank very much for this suggestion.